sumpy/e2e.py

```diff
@@ -247,6 +247,259 @@ def __call__(self, queue, **kwargs):
                 src_rscale=src_rscale, tgt_rscale=tgt_rscale,
                 **kwargs)
 
+
+class E2EFromCSRTranslationInvariant(E2EFromCSR):
```

Comment:
AFAICT, the main difference between this and `E2EFromCSR` is really that it deals with translation classes. Is that equivalent to translation invariance? If not, maybe change the name to something involving translation classes?

Reply:
Yes, they are equivalent.